test/unit/communication/test_transform_mpi.cpp

```diff
@@ -99,7 +99,7 @@ TEST_F(TransformMPITest, PromiseGuardManagement) {
     // At this point we checked if everything went as expected in the posting phase. Let's signal
     // Rank 1 that it can send back the message that will unlock IRecv.
     const int signal = 13;
-    MPI_Send(&signal, 1, MPI_INT, 1, 0, world);
+    MPI_Send(const_cast<int*>(&signal), 1, MPI_INT, 1, 0, world);
```

Comment:
As `signal` is used only here:
```suggestion
    int signal = 13;
    MPI_Send(&signal, 1, MPI_INT, 1, 0, world);
```

Reply:
Updated.